python/ray/serve/_private/router.py

```diff
@@ -781,6 +877,7 @@ async def fulfill_pending_requests(self):
             logger.exception("Unexpected error in fulfill_pending_requests.")
         finally:
             self._scheduling_tasks.remove(asyncio.current_task(loop=self._loop))
+            self.num_scheduling_tasks_gauge.set(len(self._scheduling_tasks))
```

Comment:
use `self.curr_num_scheduling_tasks`

Reply:
Sounds good, I switched to `self.curr_num_scheduling_tasks`.